### PR TITLE
feat(design): added article component examples

### PR DESCRIPTION
**Pull request by Nolan-Arnold**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

**Comment by xelaint** [comment]
Is this WIP, waiting on #892? I don't see the code previews actually implemented in the article docs.

**Comment by Nolan-Arnold** [comment]
> Is this WIP, waiting on #892? I don't see the code previews actually implemented in the article docs.

I don't believe the code previews were ever written for the article docs? If they were, they were not in the branch I split. I can write them once these PRs were merged in @xelaint .

**Comment by xelaint** [comment]
@Nolan-Arnold Isn't the article docs split into the example components so that it can be rendered using the code preview component?

**Comment by Nolan-Arnold** [comment]
@xelaint Sorry this took me awhile to get back to. So they were all already split into examples and I think what happened is Damien wrote and pushed the examples but didn't push the actual rendering of them in the html so I went ahead and just added that part myself. Now all of the component examples are being rendered with the code preview component in the latest commit92c6bd2a6d60ac55367dfdacec3e47963a5717b9.

File: apps/design-land/src/app/article/article.component.html [hunk 1]
```diff
@@ -6,59 +6,43 @@ <h2>About</h2>
...
-	<h2>This is a h2 heading.</h2>
-	<h3>This is a h3 heading.</h3>
-	<h4>This is a h4 heading.</h4>
-	<h5>This is a h5 heading.</h5>
-	<h6>This is a h6 heading.</h6>
+	<design-land-example-viewer example="article-headings"></design-land-example-viewer>
+
 
 	<h2>Article Lead</h2>
 	<p daffArticleLead>Lead is used for the opening paragraph of an article. It should be used to provide a summary or leading information for an article. Lead is a custom directive of article and is not a native element selector. To use it, add <code>daffArticleLead</code> to a paragraph (<code>&lt;p&gt;</code>)</p>
```
Comment (xelaint) [inline apps/design-land/src/app/article/article.component.html:13]: Description/usage instruction should be using the `<p>` tag without its own directive attached to it.

File: apps/design-land/src/app/article/article.component.html [hunk 2]
```diff
@@ -6,59 +6,43 @@ <h2>About</h2>
...
+	<design-land-example-viewer example="article-lead"></design-land-example-viewer>
+
 
 	<h2>Article Meta</h2>
 	<p daffArticleMeta>Meta is used if there is metadata information about your article (i.e. author name, date, etc). Lead is a custom directive of article and is not a native element selector. To use it, add <code>daffArticleMeta</code> to a paragraph (<code>&lt;p&gt;</code>)</p>
```
Comment (xelaint) [inline apps/design-land/src/app/article/article.component.html:18]: Description/usage instruction should be using the `<p>` tag without its own directive attached to it.

Comment (Nolan-Arnold) [reply apps/design-land/src/app/article/article.component.html]: Fixed in b39de07ab474a4355c1c1f0ee530875c0e9a107d

File: apps/design-land/src/app/article/article.component.html (hunk 1 above)
Comment (Nolan-Arnold) [reply apps/design-land/src/app/article/article.component.html]: Fixed in b39de07ab474a4355c1c1f0ee530875c0e9a107d

**Pull request merged**